src/main/java/online/partyrun/partyrunmatchingservice/domain/matching/entity/Matching.java

```diff
@@ -47,20 +47,12 @@ public void cancel() {
         members.forEach(member -> member.changeStatus(MatchingMemberStatus.CANCELED));
     }
 
-    public void updateMemberStatus(final String memberId, final MatchingMemberStatus status) {
-        final MatchingMember member = findMember(memberId);
-        member.changeStatus(status);
-        this.status = getMatchStatus();
+    public boolean updateStatus() {
+        this.status = generateMatchingStatus();
+        return !this.status.equals(MatchingStatus.WAIT);
```

Comment:
```suggestion
        return !this.status.isWating()
```
MatchingStatus 에 변경이 일어나면 Matching까지 영향을 받아야하나요?
예를 들어 WAIT 라는 이름이 WAITING으로 바뀐다거나..

Reply:
반영했습니다